### PR TITLE
Update style.css for Japanese Font. Issue #59

### PR DESCRIPTION
**Pull request by nukaga**
Update style.css for Japanese Font. Issue #59

### CODE REVIEW

**Comment by melchoyce** [comment]
Hey, tested this out and it looks great.

@laurelfulford Do you have any preference where language-specific styles live in the stylesheet?

**Comment by laurelfulford** [comment]
@melchoyce Adding them under the 'Typography' header makes sense to me! 

I think, if we add a lot of languages, it may be good to move the Genericons/icon font styles above them, but that's minor and we can cross that bridge when we get to it.

**Comment by melchoyce** [comment]
> Adding them under the 'Typography' header makes sense to me!

Cool. Good to merge?

> I think, if we add a lot of languages, it may be good to move the Genericons/icon font styles above them, but that's minor and we can cross that bridge when we get to it.

Hopefully we won't have any icon font styles much longer ( #206 )

**Comment by laurelfulford** [comment]
> Cool. Good to merge?

I took a closer look and I can see a couple minor formatting tweaks that will make this CSS match the rest of the file:
- On lines 553 and 563, a space should be added after `font-family:`.
- On the same lines, `Hiragino Kaku Gothic Pro` should be wrapped in double-quotes, rather than single quotes.

That's it!

> Hopefully we won't have any icon font styles much longer

Right! Good point. :)

File: style.css [hunk 1]
```diff
@@ -543,6 +543,28 @@ q:after {
...
 }
 
+/* Typography for Japanese Font */
+
+html[lang="ja"] body,
+html[lang="ja"] button,
+html[lang="ja"] input,
+html[lang="ja"] select,
+html[lang="ja"] textarea {
+	font-family:'Hiragino Kaku Gothic Pro', Meiryo, sans-serif;
```
Comment (melchoyce) [inline style.css:553]: Need a space between `font-family:` and `'Hiragino'`

File: style.css [hunk 2]
```diff
@@ -543,6 +543,28 @@ q:after {
...
+	letter-spacing: normal;
+}
+
+html[lang="ja"] h1, 
+html[lang="ja"] h2, 
+html[lang="ja"] h3,
+html[lang="ja"] h4,
+html[lang="ja"] h5,
+html[lang="ja"] h6 {
+	font-family:'Hiragino Kaku Gothic Pro', Meiryo, sans-serif;
```
Comment (melchoyce) [inline style.css:563]: Need a space between font-family: and 'Hiragino'

Comment (melchoyce) [reply style.css:563]: `Hiragino Kaku Gothic Pro` should be wrapped in `""`, not `''`

File: style.css (hunk 1 above)
Comment (melchoyce) [reply style.css:553]: `Hiragino Kaku Gothic Pro` should be wrapped in `""`, not `''`

**Review by melchoyce — CHANGES_REQUESTED** [review]

File: style.css [hunk 3]
```diff
@@ -543,6 +543,28 @@ q:after {
...
+	letter-spacing: normal;
+	font-weight: bold;
```
Comment (ntwb) [inline style.css:565]: This should be numeric, i.e. `700`, not `bold`

File: style.css [hunk 4]
```diff
@@ -543,6 +543,28 @@ q:after {
...
+html[lang="ja"] body,
+html[lang="ja"] button,
+html[lang="ja"] input,
+html[lang="ja"] select,
+html[lang="ja"] textarea {
+	font-family:'Hiragino Kaku Gothic Pro', Meiryo, sans-serif;
+	letter-spacing: normal;
+}
+
+html[lang="ja"] h1,
```
Comment (ntwb) [inline style.css:557]: There's a trailing space at the end of this line that should be removed

File: style.css [hunk 5]
```diff
@@ -543,6 +543,28 @@ q:after {
...
+html[lang="ja"] button,
+html[lang="ja"] input,
+html[lang="ja"] select,
+html[lang="ja"] textarea {
+	font-family:'Hiragino Kaku Gothic Pro', Meiryo, sans-serif;
+	letter-spacing: normal;
+}
+
+html[lang="ja"] h1, 
+html[lang="ja"] h2,
```
Comment (ntwb) [inline style.css:558]: Another trailing space at the end of this line that should be removed

**Review by ntwb — CHANGES_REQUESTED** [review]
Added a couple more coding standards issues

**Comment by melchoyce** [comment]
Hi @nukaga, do you know if you'll have time to update this PR within the next couple of days? If not, please let us know and someone else can finish it. :) Thanks!

**Comment by melchoyce** [comment]
Continuing this PR in #317.

**Pull request closed without merging**